### PR TITLE
UI of the calculator created

### PR DESCRIPTION
**Pull request by Juanse7793**
## Here I:

- Create the UI of the calculator for the math magician project

## Requirements:

- I delete all the boilerplate from CRA (text, images, styles).
- I create a directory called components.
- Inside components, I create a new calculator.js file.
- In Calculator.js, I create a React component that matches the given design
- I don't add any extra design.
- I develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- I use class-based components.
- Once the Calculator was ready, I import it and used it in the main component (App.js).
- I test it in the browser; it renders without problems.

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Dipeshtwis) [inline src/App.js]: - [x] The `App.js` component should be class-based. Please update it.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Juanse7793, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:+1: The design is looking the same as given.

## Required Changes ♻️
_Check the comments under the review._

- [x] I would recommend adding the proper title, description, and proper steps to clone and run the project locally.

```
$ git clone https://github.com/Juanse7793/Math-Magicians.git
$ cd Math-Magicians
$ npm install
$ npm start
```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Juanse7793) [reply src/App.js]: HI dear @Dipeshtwis I think that this is not a requirement, they said that all components should be class-based but I don't know if App.js is also a component (they said that in the App.js we should only import the calculator component), anyway it's done.

**Review by mmsesay — APPROVED** [review]
Hi @Juanse7793, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**